web/src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -237,9 +240,10 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
 
   handleSubmit = async () => {
     const { onSubmit } = this.props
-    const { selectedFeedbackOption, comment } = this.state
+    const { selectedFeedbackOption, comment, contactMail } = this.state
     try {
-      await this.postFeedbackData(this.getFeedbackData(selectedFeedbackOption, comment))
+      await this.postFeedbackData(
+        this.getFeedbackData(selectedFeedbackOption, `${comment}\m\nKontakt Adresse: ${contactMail}`))
```

Comment:
```suggestion
        this.getFeedbackData(selectedFeedbackOption, `${comment} Kontaktadresse: ${contactMail}`))
```
otherwise it looks quite weird as linebreaks are not working.

Reply:
@Cmd190 and I discussed this in mattermost. For now we will go with the white-space.

Reply:
@maxammann so the way it is currently?